presto-main/src/main/java/com/facebook/presto/operator/scalar/HyperLogLogFunctions.java

```diff
@@ -38,6 +39,14 @@ public static long cardinality(@SqlType(StandardTypes.HYPER_LOG_LOG) Slice seria
     @SqlType(StandardTypes.HYPER_LOG_LOG)
     public static Slice emptyApproxSet()
     {
-        return ApproximateSetAggregation.newHyperLogLog().serialize();
+        return HyperLogLog.newInstance(ApproximateCountDistinctAggregation.standardErrorToBuckets(ApproximateSetAggregation.DEFAULT_STANDARD_ERROR)).serialize();
+    }
+
+    @ScalarFunction
+    @Description("an empty HyperLogLog instance with the specified max standard error")
+    @SqlType(StandardTypes.HYPER_LOG_LOG)
+    public static Slice emptyApproxSet(@SqlType(StandardTypes.DOUBLE) double maxStandardError)
```

Comment:
Maybe an overkill: the empty HLL may have "unknown" bucket count. The bucket count remains unknown when merging with an unknown-bucket HLL or coalesced with the other one's bucket count. But given the complication, I don't have strong opinion having this.

Reply:
I agree with it, this would be a nice thing to do, but if it's too complicated we can skip it

Reply:
Yes that would be nice, but we don't currently have access to the bucket count of an HLL so I don't think it is easy yet. I opened a PR to allow this so in future this could be possible: https://github.com/airlift/airlift/pull/724